src/app/auth/register/profile-info/profile-info.component.ts

```diff
@@ -87,7 +88,7 @@ export class ProfileInfoComponent implements OnInit, OnDestroy {
     }
 
     // length is OK, let's check for proper structure
-    const r: RegExp = /([0-9]{1})|([a-z]){1}|([A-Z]){1}|([!,@#$%^*&|~`\]\[\{\}<>.\\_\-+=\(\)/?]{1})/g;
```

Comment:
Was this type deprecated or was the type just not necessary here?

Reply:
This was just was not being used